lib/app-components/addon/components/branded-navbar/template.hbs

```diff
@@ -34,6 +34,8 @@
                     {{#if @displayModerationButton}}
                         <li>
                             <OsfLink
+                                data-analytics-name='collection-moderation'
```

Comment:
Just a bit of convention, for data-analytics-names, we usually use something like `data-analytics-name='Collection moderation'`